docs/modes-of-operation.rst

```diff
@@ -206,9 +206,9 @@ must be met:
 
 
 - :ref:`setting-superslave` support must be enabled
-- The supermaster must carry a SOA record for the notified domain
-- The supermaster IP must be present in the 'supermaster' table
-- The set of NS records for the domain, as retrieved by the slave from the supermaster, must include the name that goes with the IP address in the supermaster table
+- The supermaster must carry a SOA record for the notified domain corresponding with one of the NS records for that domain
```

Comment:
This sentence was correct before you changed it - the name in the SOA is ignored.